components/collapsible-panel/collapsible-panel.js

```diff
@@ -217,6 +217,8 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 			.d2l-collapsible-panel-opener {
 				align-self: self-start;
 				margin-inline-end: var(--d2l-collapsible-panel-spacing-inline);
+				order: 1;
+				outline: none;
```

Comment:
- `order` seems to be working as expected but I saw concerns raised about it and we are not using it elsewhere in the repo. 
- Removing the outline may not be sufficient to remove focus styles from an element

Reply:
I think Danny ran into some difficulty using `order` with Safari in `d2l-list`. Would be worthwhile doing some more thorough testing of that though, if you haven't already.

The `outline: none;` should be sufficient.